e2e/suite_test.go

```diff
@@ -26,14 +25,16 @@ var (
 	gitRepositoryPort = 3000
 	hostUrl           = "localhost"
 	portSeparator     = ":"
+	localOciRegistry  = "registry"
+	localGitService   = "gitea"
 )
 
 func TestMain(m *testing.M) {
 	setupLog("starting e2e test suite")
 
 	cfg, _ := envconf.NewFromFlags()
 	testEnv = env.NewWithConfig(cfg)
-	kindClusterName = envconf.RandomName("ocm-ctrl-e2e"+time.Now().Format("2006-01-02-15-04-05"), 32)
+	kindClusterName = envconf.RandomName("ocm-ctrl-e2e-"+time.Now().Format("2006-01-02-15-04-05"), 4)
```

Comment:
Ah I didn't notice this was left in last time. Please remove the timestamp. Random already inserts randomized data. No need for the extra timestamping